### DISCUSSION TITLE
Where does funding come from?

### DISCUSSION THREAD

**Discussion by eobet** [rustdesk/rustdesk#3090] [General]
Everything about Rustdesk appears very professional for an open source project with no apparent means of donations or other means of payment. This is a bit strange, especially since the project relies of server infrastructure in order to work at all for those who don't self-host, and someone must pay those monthly bills.

How is this project funded?

**Comment by Benny7587** [comment]
Last week i made a donation via: https://ko-fi.com/rustdesk

**Reply by eobet** [reply]
According to that site, they've received 273 donations. That's not enough to finance something like this.

Does Lenovo provide the rest of the funding? The company behind this project is registered in Hong Kong, so could they be receiving any government grants?

**Comment by Benny7587** [comment]
The nice thing about RustDesk is that if you do not trust whatever government, you can self host the server. Imho that makes the solution more secure then AnyDesk and Teamviewer together.

**Comment by shivkr6** [comment]
It's VC funded.
https://oss.capital/portfolio/RustDesk/

**Comment by southerndoc** [comment]
I pay an annual subscription to RustDesk Server Pro.
